src/NHibernate/Dialect/DB2Dialect.cs

```diff
@@ -270,5 +270,34 @@ public override string ForUpdateString
 		{
 			get { return " for read only with rs"; }
 		}
+
+		#region Overridden informational metadata
+
+		public override bool SupportsEmptyInList
+		{
+			get { return false; }
```

Comment:
Can these one-line properties be written with expression syntax? I know that it'll make an inconsistency, but I'm ok with it.